src/AVFoundation/Enums.cs

```diff
@@ -749,4 +749,35 @@ public enum AVPlayerViewControllerSkippingBehavior : nint {
 		Default = 0,
 		SkipItem
 	}
+
+	// Convience enum for native string values 
+	public enum AVAssetExportSessionPreset {
+		[Mac(10,11)]
+		[Field ("AVAssetExportPresetLowQuality")]
+		LowQuality, // AVAssetExportPresetLowQuality
+		[Mac(10,11)]
+		[Field ("AVAssetExportPresetMediumQuality")]
+		MediumQuality, // AVAssetExportPresetMediumQuality
+		[Mac(10,11)]
+		[Field ("AVAssetExportPresetHighestQuality")]
+		HighestQuality, // AVAssetExportPresetHighestQuality
+		[Field ("AVAssetExportPreset640x480")]
+		Preset640x480, // AVAssetExportPreset640x480
+		[Field ("AVAssetExportPreset960x540")]
+		Preset960x540, // AVAssetExportPreset960x540
+		[Field ("AVAssetExportPreset1280x720")]
+		Preset1280x720, // AVAssetExportPreset1280x720
+		[Field ("AVAssetExportPreset1920x1080")]
+		Preset1920x1080, // AVAssetExportPreset1920x1080
+	#if !MONOMAC
+		[iOS (9,0)]
+		[Mac (10,10)]
+		[Field ("AVAssetExportPreset3840x2160")]
+		Preset3840x2160, // AVAssetExportPreset3840x2160
+	#endif
+		[Field ("AVAssetExportPresetAppleM4A")]
+		AppleM4A, // AVAssetExportPresetAppleM4A
```

Comment:
Because of the `!MONOMAC` above the numeric value would be different across platforms. You'll need to manually assign value in this case.